src/main/archetype/core/pom.xml

```diff
@@ -128,20 +128,33 @@
             <artifactId>org.apache.sling.models.api</artifactId>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
```

Comment:
When I did this, I just used:
```
     <groupId>org.junit.jupiter</groupId>
     <artifactId>junit-jupiter</artifactId>
```
This is a [shortcut for the three you've included](https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-jupiter).  It's a little shorter, but either way works.